apps/nextjs/src/app/[locale]/manage/about/page.tsx

```diff
@@ -102,6 +143,31 @@ export default async function AboutPage({ params: { locale } }: PageProps) {
   );
 }
 
+interface GenericContributorLinkCardProps {
+  name: string;
+  link: string;
+  image: string;
+}
+
+const GenericContributorLinkCard = ({
+  name,
+  image,
+  link,
+}: GenericContributorLinkCardProps) => {
+  return (
+    <AspectRatio ratio={1}>
+      <Card component="a" href={link} target="_blank" bg="dark.5" w={100}>
```

Comment:
Does this look good as well in light mode?

Reply:
Will fix 👍